### PR TITLE
Added O2 management supplies

### PR DESCRIPTION
**Pull request by Philbilly**
Added BVMs to Medics. Each gets one on spawn. 

Added Additional BVMs to MEV/FH/CCP

Added O2 Tanks to MEV/FH/CCP
  - CCP gets 4 150ml o2 tanks. 
  - FH and MEV get bigger 300ml O2 tanks. 
  - Universally gave 5 additional BVMs to facilities incase of mass cas

Updated CCP script to use only Alpha CCP script. 
 - Deleted squad name in Alpha CCP built notif
 - Removed squad affix from Alpha CCP marker
 - Removed Squad medic variables. 
 - Added "ComMed" Variable instead for all medics to use the same script. 
-  Resisted urge to track down drunk kenji for constantly talking over everyone.

### CODE REVIEW

**Comment by tanaaKa** [comment]
Have you tested this locally?

**Comment by Philbilly** [comment]
> Have you tested this locally?

I did for a single medic. Totally forgot about not being able to reuse variables though. 
Added additional variables. Tested them all locally with no issues. 

Also needs a once over in MP just to make sure it works as intended still, but everyones playing Baldur's gate lmfao.

File: addons/framework/buildccp/initmed.sqf [hunk 1]
```diff
@@ -43,7 +47,7 @@ if (_playerClass in _medics) then {
 		[player] call tnk_add_med_mo;
 	};
 	//Alpha medics
-	if (_playerClass in ["BLU_ComMed","OPF_ComMed","IND_ComMed"]) then {
+	if (_playerClass in ["BLU_ComMed","BLU_ComMed1","BLU_ComMed2","BLU_ComMed3","BLU_ComMed4","OPF_ComMed","OPF_ComMed1","OPF_ComMed2","OPF_ComMed3","OPF_ComMed4","IND_ComMed","IND_ComMed1","IND_ComMed2","IND_ComMed3","IND_ComMed4"]) then {
```
Comment (tanaaKa) [inline addons/framework/buildccp/initmed.sqf]: Change this line to check if playerClass is in the already pre-defined array:
```
if (_playerClass in _medics) then {
  ...
};
```

**Review by tanaaKa — CHANGES_REQUESTED** [review]

File: addons/framework/misc/welcome/fn_initWelcome.sqf [hunk 2]
```diff
@@ -48,38 +48,38 @@ switch (_role) do {
...
+		Always stick with your color team and engage any armor that is deemed a threat squad. Always remember to yell 'CLEAR BACKBLAST!' and 'ROCKET' prior to firing your launcher.";
 	};
 	case "rifleman_02":
 	{
 		_roleT = "Grenadier";
 		_brief = "";
 		_roleNotes = "You are a 40mm force multiplier for your fireteam with extra grenades to suppress the enemy or provide smoke for dangerous crossings.<br/><br/>
 		
-		Always stick with your color team and identify when your fireteam will need 40mm support in the form of smokes or HE.";
+		Always stick with your color team and identify when your team will need 40mm support in the form of smokes or HE.";
```
Comment (tanaaKa) [inline addons/framework/misc/welcome/fn_initWelcome.sqf:67]: We need a `rifleman_03` case, which is CLS.

**Review by tanaaKa — CHANGES_REQUESTED** [review]

File: addons/framework/buildccp/initmed.sqf [hunk 3]
```diff
@@ -46,51 +49,14 @@ if (_playerClass in _medics) then {
 		[player] call tnk_add_med_mo;
 	};
 	//Alpha medics
-	if (_playerClass in ["BLU_ASLM","OPF_ASLM","IND_ASLM"]) then {
+	if (_playerClass in _medics) then {
```
Comment (tanaaKa) [inline addons/framework/buildccp/initmed.sqf]: Looks good. Fix the indentation/whitespace here and we can merge

**Review by tanaaKa — CHANGES_REQUESTED** [review]

File: addons/framework/buildccp/initmed.sqf (hunk 3 above)
Comment (Philbilly) [reply addons/framework/buildccp/initmed.sqf]: Did some testing, and after running into issues convened with patman and found an issue with classes being private. 

Fixed in newest commit. 

Everything is locally tested, working, and throwing no errors. 

Should hopefully be good to go

**Pull request merged**